java/bogdana_maksiuta/src/com/zoolatech/lecture1/tasks/_1/ConverterMiToKm.java

```diff
@@ -0,0 +1,27 @@
+package com.zoolatech.lecture1.tasks._1;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts numbers of miles as a floating point number,
+ * converts it to kilometers and outputs the result.
+ *         a. Input: 2.5
+ *            Output: 4.0233603
+ */
+public class ConverterMiToKm {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        float miles = scanner.nextFloat(); //input number
+//        float miles = 2.5F; // for testing
+        if (miles > 0) {
+            converter(miles);
+        } else {
+            throw new IllegalArgumentException("Wrong number: " + miles);
+        }
+    }
+    public static void converter (float miles) {
+        final float METRIC_EQUIVALENT = (float) 1.609344;
```

Comment:
Instead of casting, you can simply add to the literal the letter `F`.

Reply:
fixed